pages/agent/v3/macos.md

```diff
@@ -17,6 +17,8 @@ Then configure your [agent token](/docs/agent/v3/tokens):
 sed -i '' "s/xxx/INSERT-YOUR-AGENT-TOKEN-HERE/g" "$(brew --prefix)"/etc/buildkite-agent/buildkite-agent.cfg
 ```
 
+In the above command, you only need to replace `INSERT-YOUR-AGENT-TOKEN-HERE` and then run the command.
+
```

Comment:
Rather than adding a new line here, I think it would be good to update the lead-in sentence for the command. Thinking:

```
Then configure your [agent token](/docs/agent/v3/tokens) by replacing `INSERT-YOUR-AGENT-TOKEN-HERE` with your agent token and running:
```

What do you think?

Reply:
100% agree, that's a much better place for it. The end goal is that any customer that reads this message understands they _only_ need to replace `INSERT-YOUR-AGENT-TOKEN-HERE` and nothing else within that command.

Reply:
I've added these changes in now and removed my original.